guides/security/samesite-protection.md

```diff
@@ -0,0 +1,28 @@
+# SameSite protection
```

Comment:
This file has to be added to the `SUMMARY.md` file, otherwise it will not show up to the navigation and will also not be available via link

Reply:
Done